doc/api/fs.md

```diff
@@ -3598,6 +3598,51 @@ that represent files will be deleted. The permissive behavior of the
 `recursive` option is deprecated, `ENOTDIR` and `ENOENT` will be thrown in
 the future.
 
+## `fs.rm(path[, options], callback)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* `options` {Object}
+  * `force` Ignore errors
```

Comment:
nit: I think we might want to be more  specific about the types of errors it ignores, from the `man` page:

> Ignore nonexistent files and missing operands

☝️ perhaps use that as a basis.